flink-jpmml-examples/README.md

```diff
@@ -18,11 +18,11 @@ assembly
 
 3) Run the examples. If you want full predictions:
 ```
-./path/to/bin/flink run -c io.radicalbit.examples.EvaluateKmeans --model path/to/pmml/model.pmml --output /path/to/output
+./path/to/bin/flink run -c io.radicalbit.examples.EvaluateKmeans path/to/flink-jpmml/flink-jpmml-examples/target/scala-2.11/flink-jpmml-examples-assembly-0.6.0-SNAPSHOT.jar --model path/to/pmml/model.pmml --output /path/to/output
```

Comment:
I think  `</path/to/project/root>/flink-jpmml-examples/target/scala-2.11/flink-jpmml-examples-assembly-0.6.0-SNAPSHOT.jar` is better in both cases. I believe also `scala-2.11` is correct only if users assembly the project with scala 2.11